math/OpenBLAS/Portfile

```diff
@@ -205,6 +205,18 @@ if {${os.platform} eq "darwin" && ${os.major} >= 23} {
     }
 }
 
+# cmake build uses CMAKE_SIZEOF_VOID_P to detect 32 or 64 bit buils,
```

Comment:
I don’t get what you mean by ‘when cmake is used from wrong arch’ ?

this really strikes me as something the OpenBLAS CMake build should handle itself…

Reply:
Perhaps the physical CPU of the buildbot does not support some insns, which are being used by clang on wrong assumptions. We had some failures on 10.6 Intel (and I think 10.8 too) due to this sort of problem.

Reply:
It is possible it's something specific to running as a VM. it's also possible it's related to the kernel, i.e. does the 64bit 10.6 OS use a 64 bit or 32 bit kernel ? This is something you should check with @ryandesign .

It's not ideal to commit this to master just on a punt that it will perhaps help, without being able to validate this first. So I am not keen to merge this until someone whom is affect by this is able to confirm it works. Even then, I am still not sure this is the right way to go about addressing this...

Reply:
@cjones051073 well.. this PR fixes universal build as well, and with some luck fixes the build bot. I here kindly ask you to merge it.

Anyway, I can't test it on my clean 10.6 without any hacks because I can't reproduce it.

I'm using Parallels to run it, maybe VmWare runs it differently some how, or VM has some hacks. I really don't know.